src/mount/mount-tool.c

```diff
@@ -263,13 +268,21 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_action = ACTION_LIST;
                         break;
 
+                case ARG_UMOUNT:
+                        arg_action = ACTION_UMOUNT;
+                        break;
+
                 case '?':
                         return -EINVAL;
 
                 default:
                         assert_not_reached("Unhandled option");
                 }
 
+        if (strstr(program_invocation_short_name, "systemd-umount")) {
```

Comment:
Single line if blocks without {} please, see CODING_STYLE. Also I think this bit should be moved before the getopt call loop, so that the command line switches always takr precedence over the implied default derived from argv0